### ISSUE TITLE
Could not load SQLite-net on Xamarin.Forms UWP

### ISSUE THREAD

**Issue by obache** [praeclarum/sqlite-net#627]
# How to reproduce
1. Create new project with Visual Studio 2017
2. Select "Cross Platform App (Xamarin)" template
3. configure as "UI" with "Xamarin.Forms", "Shareing Code" with "PCL"
4. open Solution NuGet package manager
5. install `sql-net-pcl` package and update other packages too
6. Put followings into "App" constructor in App.xaml.cs 
``` c#
            var databasePath = Path.GetRandomFileName();
            var conn = new SQLiteAsyncConnection(databasePath);
```
7. Start App.UWP project debugging
8. `System.IO.FileLoadException` exception will be thrown
> System.IO.FileLoadException: 'Could not load file or assembly 'SQLite-net, Version=1.4.118.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)'

# Environment
- Visual Studion 15.3.3
- NuGet 4.3.1
- Xamarin 4.6.0.299
- Xamarin.Forms 2.3.4.270
- sqlite-net-pcl 1.14.118

# Additional Info
- With sqlite-net-pcl 1.3.3, it worked well.
- Android project works well even with 1.14.118

**Comment by blue-toe** [comment]
I have exactly the same problem. iOS also loads fine.

**Comment by mesta1** [comment]
Have the same problem. 
When adding the project manually instead of using NuGet it works, so it seems just a matter of updating the NuGet packages with the latest commits.

**Comment by jerry2007** [comment]
+1

**Comment by AndreiMisiukevich** [comment]
Correct "Additional info", @obache )

sqlite-net-pcl sqlite-net-pcl 1.**4**.118 instead of sqlite-net-pcl 1.14.118

**Comment by obache** [comment]
@AndreiMisiukevich Thanks and fixed.

**Comment by obache** [comment]
It works well using .NET Standard instead of PCL. with Xamarin.Forms 2.4.0.282.

**Comment by deckertron9000** [comment]
@obache So it works on UWP if you use .NET Standard projects instead of PCL?

**Comment by obache** [comment]
@deckertron9000 Yes, I've just upgraded my project to  .Net Standard 1.3 project, and it works well even on UWP.

**Comment by lukealderton** [comment]
I upgraded the package within my project from 1.4.118 to 1.5.166-beta and this issue was no longer present.

**Comment by smedvedev** [comment]
When release with this fix is planned?

**Comment by luminelx64** [comment]
@lukealderton I upgraded to 1.5.166-beta but it didn't solve the issue.
